src/sentry/middleware/ratelimit.py

```diff
@@ -115,6 +119,11 @@ def process_view(
                                 limit=request.rate_limit_metadata.limit,
                                 window=request.rate_limit_metadata.window,
                             )
+                            if request.rate_limit_metadata.rate_limit_type
+                            == RateLimitType.FIXED_WINDOW
+                            else DEFAULT_CONCURRENT_ERROR_MESSAGE.format(
+                                limit=request.rate_limit_metadata.concurrent_limit
+                            )
```

Comment:
nit: I sort of feel like this would be easier to read as a if else block now

```py
if request.rate_limit_metadata.rate_limit_type == RateLimitType.FIXED_WINDOW: 
  response = DEFAULT_ERROR_MESSAGE.format(
    limit=request.rate_limit_metadata.limit,
    window=request.rate_limit_metadata.window,
  )
else:
  response = DEFAULT_CONCURRENT_ERROR_MESSAGE.format(
    limit=request.rate_limit_metadata.concurrent_limit
  )
```